install/deploy.sh

```diff
@@ -48,7 +48,7 @@ fi
 
 export KUBECONFIG=${WD}/local-kubeconfig.yaml
 kubectl create namespace istio-system || true
-kubectl apply -f $GOPATH/src/istio.io/istio/manifests/charts/base/crds/
+kubectl apply -f $GOPATH/istio/manifests/charts/base/crds/
```

Comment:
This isn't how GOPATH works; the old version is correct. That being said, GOPATH is not really the modern way to do this...

Reply:
I am reverting my changes for this PR. As I understood GOPATH is the default workspace location. I think we may need to take location of Istio repo as input (or maybe location of base crds) from user instead of using GOPATH.

Reply:
Can we at least default to `$GOPATH/src/istio.io/istio`? This is where most users have it, since it was required in older go versions

Reply:
We are defaulting to $GOPATH/src/istio.io/istio only in latest changes.